carbon-design-system/carbon-addons-iot-react
# fix(imageTile): Improve ImageTile regex

Closes #2010 

**Summary**

- Improve regular expression to extract file name without extension
 
**Change List (commits, features, bugs, etc)**

- changed regex

**Acceptance Test (how to verify the PR)**

- check ImageGallery story on deploy preview to make sure file names without extensions are still being rendered correctly. 
- check that CodeQL doesn't throw up a warning about an inefficient regex
